src/server/server.js

```diff
@@ -141,7 +141,7 @@ if (isDevelopment) {
     }),
     {}
   )
-  getDatasetForRequest = (req) => datasetBySubdomain[req.subdomains[0]]
+  getDatasetForRequest = () => datasetBySubdomain['ibd']
```

Comment:
I wouldn't get too side tracked on this, but I can see there maybe being a more generalizable solution here where we can set an environment variable that tells the browser which dataset you want to run with, instead of the typical production build that's doing the hostname based loading.